rdir/rdir.c

```diff
@@ -1159,7 +1192,12 @@ _route_admin_clear(struct req_args_s *args, const char *volid, const char *all)
 		return _reply_common_error(args->rp, err);
 
 	gint64 nb_removed = 0;
-	err = _db_admin_clear(volid, oio_str_parse_bool(all, FALSE), &nb_removed);
+	gint64 nb_repaired = 0;
+	gint64 errors = 0;
+	err = _db_admin_clear(volid, oio_str_parse_bool(all, FALSE),
```

Comment:
We should avoid functions with too many arguments.
In this case, all the output variables could be packed into a structure.
(this is not blocking)